apollo-router/src/axum_factory/axum_http_server_factory.rs

```diff
@@ -414,6 +415,17 @@ where
     Ok(ListenAddrAndRouter(listener, route))
 }
 
+async fn metrics_handler<B>(request: Request<B>, next: Next<B>) -> Response {
+    let resp = next.run(request).await;
+    u64_counter!(
+        "apollo.router.operations",
+        "The number of graphql operations performed by the Router",
+        1,
+        "http.response.status_code" = resp.status().as_u16() as i64
```

Comment:
Should that be `as u64`? It's a `u64_counter`
(I just spotted it's a copy and paste of the original, but now I'm wondering about it...)

Reply:
Although monotonic counters are `u64` attributes on metrics are `i64`